qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -109,7 +109,6 @@ def set_cmd_text_command(self, text, space=False, append=False):
             append: If given, the text is appended to the current text.
         """
         args = split.simple_split(text)
-        args = runners.replace_variables(self._win_id, args)
```

Comment:
That line was there because the variable replacement would otherwise only happen if `{url}` or `{url:pretty}` were the full command text, but that's no longer a problem. I should remove the two lines surrounding it as well, I missed that.